roles/hammerdb/templates/db_workload.yml

```diff
@@ -0,0 +1,29 @@
+---
+
+apiVersion: batch/v1
+kind: "job"
+metadata:
+  name: "{{ meta.name }}-hammerdb-workload"
```

Comment:
following pr #191 any resources created should make use of the `trunc_uuid` var appended to name and label.

Reply:
fixed